docs/testnet/join-testnet-tutorial.md

```diff
@@ -0,0 +1,292 @@
+## Join the Interchain-Security Testnet
+This guide contains the instructions for joining a Interchain-Security Testnet. You can find the instructions to setup a IS-testnet from scratch [here](https://github.com/sainoe/IS-testnet/blob/main/is-testnet-tutorial.md). 
+
+---
+
+### Prerequesites
+- Go 1.18+
+- Interchain-Security binaries
+- jq
+
+#### Install Go
+
+```
+curl -OL https://golang.org/dl/go1.18.4.linux-amd64.tar.gz
+
+sudo tar -C /usr/local -xvf go1.18.4.linux-amd64.tar.gz
+
+export PATH=$PATH:/usr/local/go/bin
+
+export PATH=$PATH:$(go env GOPATH)/bin
+```
+
+---
+
+#### Install Interchain-Security binaries
+
+```
+git clone https://github.com/cosmos/interchain-security.git
+
+cd interchain-security
+
+make install
+```
+
+---
+
+#### Install jq
```

Comment:
Move to `start-testnet-tutorial.md` or even better, remove and add link to installation guide in `start-testnet-tutorial.md`.